### PR TITLE
Two small miscellaneous tests

### PR DESCRIPTION
**Pull request by kshyatt**

### CODE REVIEW

File: test/sysinfo.jl [hunk 1]
```diff
@@ -7,6 +7,8 @@ sprint(Base.Sys.cpu_summary)
 @test Base.Sys.uptime() > 0
 Base.Sys.loadavg()
 
+@test !("fakename" ∈ keys(ENV))
```
Comment (tkelman) [inline test/sysinfo.jl]: you never know... maybe something longer and more unlikely?

Comment (kshyatt) [inline test/sysinfo.jl]: `the_test_name_is_a_lie`?

Comment (davidanthoff) [inline test/sysinfo.jl]: You could create a GUID with https://www.guidgenerator.com/ and use that as the string.

Comment (dpsanders) [inline test/sysinfo.jl]:
```
julia> Base.Random.uuid1()
46102602-1efd-11e6-3a26-c5f5005a8b83
```

Comment (dpsanders) [inline test/sysinfo.jl]: (I just searched for Julia UUID and found https://github.com/JuliaArchive/UUID.jl)

Comment (simonster) [inline test/sysinfo.jl]: Maybe something with an = sign in it, since that's not a valid environmental variable name (at least on POSIX).

**Comment by kshyatt** [comment]
Rebased and followed @simonster's suggestion.

**Pull request merged**